<?xml version="1.0"?>
<launch>

<node name="path_planner" pkg="motion_controller" type="path_planner_main.py" respawn="false" output="screen"/>

</launch>